src/coreComponents/finiteVolume/StencilBase.hpp

```diff
@@ -82,16 +85,19 @@ class StencilWrapperBase : public TRAITS
    * @param elementRegionIndices The container for the element region indices for each point in each stencil
    * @param elementSubRegionIndices The container for the element sub region indices for each point in each stencil
    * @param elementIndices The container for the element indices for each point in each stencil
+   * @param partialWeights The container for the partial weights for each point in each stencil
    * @param weights The container for the weights for each point in each stencil
    */
   StencilWrapperBase( typename TRAITS::IndexContainerType const & elementRegionIndices,
                       typename TRAITS::IndexContainerType const & elementSubRegionIndices,
                       typename TRAITS::IndexContainerType const & elementIndices,
+                      typename TRAITS::WeightContainerType const & partialWeights,
                       typename TRAITS::WeightContainerType const & weights ):
     m_elementRegionIndices( elementRegionIndices.toViewConst() ),
     m_elementSubRegionIndices( elementSubRegionIndices.toViewConst() ),
     m_elementIndices( elementIndices.toViewConst() ),
-    m_weights( weights.toViewConst() )
+    m_partialWeights( partialWeights.toView() ),
```

Comment:
are we going to need `partialWeights` also for the `CellElementTPFAStencil`? If not, I think that we could define it directly only in the stencils involving fractures.

Reply:
I may be wrong but I think they all use the same interface. And also we thought we may keep it this way in case it's useful for large deformation. But I am open to limit it to fracture stencils if it doesn't require too many changes.